src/Controller/QuestionInstanceController.php

```diff
@@ -71,15 +71,15 @@ public function getQuestionInstance(Request $request, array $requestAttributes)
         $quizInstanceId = $this->session->get('quizInstanceId');
         $questionInstanceIndex = $requestAttributes['id'];
         $criteria = new Criteria(['quiz_instance_id' => $quizInstanceId], [], $questionInstanceIndex - 1, 1);
-        $question = $this->questionInstanceRepository->findBy($criteria);
-        $answer = $this->answerInstanceRepository->getAnswers($question[0]->getId());
+        $questionSearchResult = $this->questionInstanceRepository->findBy($criteria)->getItems();
+        $answer = $this->answerInstanceRepository->getAnswers($questionSearchResult[0]->getId());
```

Comment:
- you could have ->getFirstItem()
- if there is no first item, the call will fail